molter/help.py

```diff
@@ -0,0 +1,197 @@
+import functools
+import logging
+
+import dis_snek
+from dis_snek import Embed
+from dis_snek.ext.paginators import Paginator
+
+import molter
+
+__all__ = ("HelpCommand",)
+
+log = logging.getLogger(dis_snek.const.logger_name)
+
+
+class HelpCommand:
+    show_hidden: bool
+    """Should hidden commands be shown?"""
+    show_disabled: bool
+    """Should disabled commands be shown?"""
+    run_checks: bool
+    """Should commands be checked if they can be run by the help command user?"""
+    show_self: bool
+    """Should this command be shown in the help message?"""
+    show_params: bool
```

Comment:
This should be renamed to `show_usage` - see a later comment for more information.